docs/documentation/server_admin/topics/realms/proc-using-admin-console.adoc

```diff
@@ -15,7 +15,7 @@ For example, for localhost, use this URL: http://localhost:8080{kc_admins_path}/
 .Login page
 image:images/login-page.png[Login page]
 
-. Enter the username and password you created on the Welcome Page or the `add-user-keycloak` script in the bin directory.
```

Comment:
Does it make sense to reference this section in our guides as a replacement https://www.keycloak.org/server/configuration#_creating_the_initial_admin_user ?

Reply:
Thank you for the comment. I added the link to the "Creating the initial admin user" guide.